### PR TITLE
fix move all compass to its own file

### PR DESCRIPTION
**Pull request by mixcas**
Please, build and run on ur devices and test this branch.

### CODE REVIEW

**Comment by varffs** [comment]
It don't work. Actually worse for me not sure why I can no longer reach the goal the page crashes:

image uploaded from ios

**Comment by mixcas** [comment]
I've fixed a bug inside the setupLoop that was leaving the loop empty on the first run. Plz check if this fixes.

**Comment by varffs** [comment]
Pretty sure that commit is missing the source files just has built ones: https://github.com/interglobalvision/walking-game/pull/145/commits/9921d0841525feb03d7716e855320907f2599964

**Comment by mixcas** [comment]
@varffs no the modified file is `cordova/www/js/src/game.js`

File: cordova/www/pages/compass/compass.js [hunk 1]
```diff
@@ -186,12 +186,24 @@ Compass = {
       _this.mapFloorPos = 0;
     }
 
+    // if mapFloorPos is grater than 100, we set it to 100
+    // this keeps the floor from sliding off screen
+    if (_this.mapFloorPos >= 100) {
```
Comment (varffs) [inline cordova/www/pages/compass/compass.js]: if value = x set to x is a bit unneeded i think

Comment (mixcas) [inline cordova/www/pages/compass/compass.js]: yeah, my mistake

**Pull request merged**